lib/galaxy/config/schemas/uwsgi_schema.yml

```diff
@@ -0,0 +1 @@
+../../../../lib/galaxy/webapps/uwsgi_schema.yml
```

Comment:
Shouldn't this also be moved here in `lib/galaxy/config/schemas/` instead of symlinked?

Reply:
And then create symlinks to `lib/galaxy/webapps/reports/uwsgi_schema.yml` and `lib/tool_shed/webapp/uwsgi_schema.yml` ?